src/util/segmentation/operations/fillInsideCircle.js

```diff
@@ -0,0 +1,30 @@
+import { getBoundingBoxAroundCircle } from '../boundaries';
+import { getEllipse, pointInEllipse } from '../../ellipse';
+
+export default function fillInsideCircle(
+  points,
+  segmentationData,
+  evt,
+  labelValue = 1
+) {
+  const { width } = evt.detail.image;
+  const [topLeft, bottomRight] = getBoundingBoxAroundCircle(evt);
+  const [xMin, yMin] = topLeft;
+  const [xMax, yMax] = bottomRight;
+
+  for (let x = xMin; x < xMax; x++) {
+    for (let y = yMin; y < yMax; y++) {
+      const inside = pointInEllipse(
+        getEllipse(evt.detail.handles.start, evt.detail.handles.end),
```

Comment:
Why is this `getEllipse` inside the loop? This could be run thousands of times, there is no reason that this function call needs to be inside the loop since the ellipse never changes once you've reached this function. Move it outside.